### DISCUSSION TITLE
Can't Find SnapShots

### DISCUSSION THREAD

**Discussion by Sptrr99** [mrlt8/docker-wyze-bridge#492] [Q&A]
All,

I've setup everything as it should be, but I'm not seeing the snapshots in the container.  I can view the web stream of my camera and I do have an SD card installed.  I've tried various env variables, but I can't seem to get any snaps.

Thanks.

**Comment by Sptrr99** [comment]
I fixed it myself.  I was using Portainer and added the env variables in the stack, but hadn't updated the docker-compose.yml file to use the new variables.

**Reply by tensiondriven** [reply]
Which variables did you use?  I'm having trouble with the `/photo` endpoint, maybe your config would give me some pointers.
